README.md

```diff
@@ -37,6 +37,8 @@ This will automatically resolve the agent module, bundle the agent artifact in t
 
 By default, sbt-javaagent will only add an agent to distributions. Agents can be optionally enabled for compile, run, or test.
 
+> **Note**: starting from v0.2.0, if the agent is only in `test` scope, it won't be added do distributions. You can set the scope to `dist;test` to retain the previous behaviour.
```

Comment:
:information_source: I assume this PR would be released as a new minor, feel free to change this line if you prefer to release it as bugfix.

Reply:
We need to release at least a new minor version because in main branch we also dropped support for SBT 0.13.
Will take a look at this pr soon